suite_cam6.xml

```diff
@@ -0,0 +1,361 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="cam6" version="1.0">
+  <group name="physics_bc">
+    <!--cam6:  deep=ZM, shallow=CLUBB, macrop=CLUBB, microp=MG2, radiation=RRTMG, chem=trop_mam4 -->
+    <time_split>
+      <scheme>physics_state_check</scheme> <!-- Only if state_debug_checks is True. This scheme only looks for infs/nans, and kills the model if found. -->
+      <scheme>clybry_fam_adj</scheme>      <!-- Only matters if chemistry package with Cly and Bry is used -->
+      <!-- Negative moisture/tracer check -->
+      <!-- **************************** -->
+      <scheme>qneg3</scheme>               <!-- This scheme modifies the state directly.  Combine with state check? -->
+      <!-- **************************** -->
+      <scheme>physics_state_check</scheme>      <!-- Only if state_debug_checks is True. This scheme only looks for infs/nans, and kills the model if found. -->
+      <scheme>diag_state_b4_phys_write</scheme> <!-- Diagnostics -->
+      <!-- energy and momentum fixer -->
```

Comment:
Was check_tracers_init intentionally left out here?  I see that it was moved to the "ac" section.  Are the tracers the same value before and after coupling?

Reply:
Whoops, didn't mean to leave the `check_tracers_init` call out here.  I can add it back in.   

However, I also realized that this scheme seems more like a "timestep_init" routine, and so may not need to be in this file at all.  Any thoughts @gold2718?

Reply:
I would say 'probably' but I would need to look carefully to make sure that this routine only ever needs to be called at the beginning of a physics timestep. I think this may be part of a larger discussion we need to have about CAM physics. For historical purposes, we accumulate tracer (constituent) update instead of tendencies. Moving forwards, I see three options we (AMP) should explore:
* Keep the current scheme (i.e., only track updated quantities)
* Move to accumulating tendencies (for passing back to the dycore)
* Have CAM accumulate the type of tracer feedback required by the dycore (e.g., updated quantity for FV, tendency for SE).

For now, remove it (i.e., think of it as becoming a timestep_init piece of some tracer scheme which is always present or something managed by CAM instead of the CCPP) and add it to the list of things we need to discuss.

Reply:
It looks like the `check_tracers_init` routine only computes the initial vertical integrals of the tracer quantities, and zeros out the associated boundary fluxes.  I could imagine this might require running the subroutine at the beginning of each physics group (e.g. `physics_bc` and `physics_ac`), as well as after any subroutine that modifies the vertical layering (e.g. pdel), but I don't see any other reason why it couldn't be run at the beginning of the model timestep, so I have removed it for now.

Also, I will create an issue for this repo to indicate that we need to discuss what quantity or quantities we will actually accumulate over the physics timestep, and thus pass on to the dycore.

Reply:
`check_tracers_init` is the "timestep_init" routine for the `check_tracers_chng` scheme, which is included in multiple locations in this file.  So it won't be forgotten if removed, as it will be needed once we create the CCPP-ized version of `check_tracers_chng`.